### PR TITLE
updated information schema queries to return only tables

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: app/scripts/controllers/tables.js [hunk 1]
```diff
@@ -251,8 +251,8 @@ angular.module('tables', ['stats', 'sql', 'common', 'tableinfo', 'events'])
             // Table Schema
             var tableStmt = 'SELECT column_name, data_type, is_generated, generation_expression ' +
               'FROM information_schema.columns ' +
-              'WHERE table_schema = ? AND table_name = ?';
-            SQLQuery.execute(tableStmt, [tableSchema, tableName], false, false, false, false)
+              'WHERE table_type = ? AND table_schema = ? AND table_name = ?';
```
Comment (chaudum) [inline app/scripts/controllers/tables.js]: `table_type` does not exist in `information_schema.columns`

File: app/scripts/controllers/tables.js [hunk 2]
```diff
@@ -251,7 +251,7 @@ angular.module('tables', ['stats', 'sql', 'common', 'tableinfo', 'events'])
             // Table Schema
             var tableStmt = 'SELECT column_name, data_type, is_generated, generation_expression ' +
               'FROM information_schema.columns ' +
-              'WHERE table_schema = ? AND table_name = ?';
+              'WHERE AND table_schema = ? AND table_name = ?';
```
Comment (chaudum) [inline app/scripts/controllers/tables.js]: ~~AND~~

Comment (Autophagy) [inline app/scripts/controllers/tables.js]: `WHERE AND`? Is something missing here? :confused:

Comment (chaudum) [reply app/scripts/controllers/tables.js]: I forget to remove the AND because I had an additional filter before that was not necessary

Comment (chaudum) [reply app/scripts/controllers/tables.js]: pushed fixup (same commit)

**Pull request merged**